docuilib/src/theme/ReactLiveScope/index.js

```diff
@@ -1,33 +1,78 @@
 import React from 'react';
+import products from '../../assets/data/products';
 import {Colors} from 'react-native-ui-lib/style';
-import {Button, Image, Text, TouchableOpacity, View} from 'react-native-ui-lib/core';
+import {BorderRadiuses, Button, Image, Spacings, Text, TouchableOpacity, View} from 'react-native-ui-lib/core';
 import ActionBar from 'react-native-ui-lib/actionBar';
+import Assets from 'react-native-ui-lib/assets';
+import Card from 'react-native-ui-lib/card';
+import Carousel from 'react-native-ui-lib/carousel';
 import Checkbox from 'react-native-ui-lib/checkbox';
 import Chip from 'react-native-ui-lib/chip';
+import Constants from 'react-native-ui-lib/constants';
+import Drawer from 'react-native-ui-lib/drawer';
+import Icon from 'react-native-ui-lib/icon';
+import Incubator from 'react-native-ui-lib/incubator';
+import MaskedInput from 'react-native-ui-lib/maskedInput';
 import RadioButton from 'react-native-ui-lib/radioButton';
 import RadioGroup from 'react-native-ui-lib/radioGroup';
 import SegmentedControl from 'react-native-ui-lib/segmentedControl';
+import SortableGridList from 'react-native-ui-lib/sortableGridList';
+import SortableList from 'react-native-ui-lib/sortableList';
 import Switch from 'react-native-ui-lib/switch';
 import TextField from 'react-native-ui-lib/textField';
+import * as Utils from './Utils';
+
+Assets.loadAssetsGroup('icons.demo', {
+  // chevronDown: require('../../assets/icons/chevronDown.png').default,
+  chevronRight: require('../../assets/icons/chevronRight.png').default,
+  // add: require('../../assets/icons/add.png').default,
+  // camera: require('../../assets/icons/cameraSelected.png').default,
+  // close: require('../../assets/icons/close.png').default,
+  // dashboard: require('../../assets/icons/dashboard.png').default,
+  drag: require('../../assets/icons/drag.png').default
+  // image: require('../../assets/icons/image.png').default,
+  // plus: require('../../assets/icons/plus.png').default,
+  // refresh: require('../../assets/icons/refresh.png').default,
+  // search: require('../../assets/icons/search.png').default,
+  // settings: require('../../assets/icons/settings.png').default,
+  // share: require('../../assets/icons/share.png').default,
+  // info: require('../../assets/icons/info.png').default,
+  // exclamation: require('../../assets/icons/exclamationFillSmall.png').default
+});
+
+Assets.data = {products};
 
 // Add react-live imports you need here
 const ReactLiveScope = {
```

Comment:
organize the exports, export first the general namespaces like Data, Playground and then the list of components